browser/components/MarkdownEditor.js

```diff
@@ -213,6 +213,8 @@ class MarkdownEditor extends React.Component {
     if (!(editorFontSize > 0 && editorFontSize < 101)) editorFontSize = 14
     let editorIndentSize = parseInt(config.editor.indentSize, 10)
     if (!(editorFontSize > 0 && editorFontSize < 132)) editorIndentSize = 4
+    let editorLineNumber = config.editor.lineNumber
```

Comment:
I don't think the name `editorLineNumber` is appropriate because it's a boolean.

Reply:
Is 'editorShowLineNumbers` better? Or just `showLineNumbers`? Or what do you propose?

Reply:
`isLineNumberShown`? Umm, I'm not sure it's better...

Reply:
I don't think that fits the plural form (`lineNumbers`) you suggested...

Reply:
I often use `is` even if the var name is plural, and the prefix `is` describes it's a boolean value. `isLineNumbersShown`...?

Reply:
OK then. I meant just the language (English) thing. Other than that, I think `isLineNumbersShown` is fine.